studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx

```diff
@@ -1,28 +1,246 @@
+                ready to use.
+              </p>
+            </div>
+          </div>
+
+          <div className="grid grid-cols-12 flex gap-4">
+            <div className="col-span-7 space-y-8 flex flex-col justify-center">
+              <div className="space-y-2">
+                <h3 className="text-xl text-scale-1200">
+                  Get started by building out your database
+                </h3>
+                <p className="text-base text-scale-1100">
+                  Start building your app by creating tables and inserting data. Our table editor
+                  makes Postgres as easy to use as a spreadsheet, but there's also our SQL editor if
+                  you need something more.
+                </p>
+              </div>
+              <div className="flex items-center space-x-2">
+                <Link href={`/project/${ref}/editor`}>
+                  <a>
+                    <Button
+                      type="default"
+                      icon={
+                        <SVG
+                          src="/img/table-editor.svg"
+                          style={{ width: `${14}px`, height: `${14}px` }}
+                          preProcessor={(code) =>
+                            code.replace(/svg/, 'svg class="m-auto text-color-inherit"')
+                          }
+                        />
+                      }
+                    >
+                      Table editor
+                    </Button>
+                  </a>
+                </Link>
+                <Link href={`/project/${ref}/sql`}>
+                  <a>
+                    <Button
+                      type="default"
+                      icon={
+                        <SVG
+                          src="/img/sql-editor.svg"
+                          style={{ width: `${14}px`, height: `${14}px` }}
+                          preProcessor={(code) =>
+                            code.replace(/svg/, 'svg class="m-auto text-color-inherit"')
+                          }
+                        />
+                      }
+                    >
+                      SQL editor
+                    </Button>
+                  </a>
+                </Link>
+                <Link href="https://supabase.com/docs/guides/database">
+                  <a>
+                    <Button type="default" icon={<IconExternalLink size={14} />}>
+                      About Database
+                    </Button>
+                  </a>
+                </Link>
+              </div>
+            </div>
+            <GetStartedHero />
+          </div>
+
+          <div className="flex flex-col justify-between h-full space-y-6">
+            <div className="space-y-2">
+              <h3 className="text-xl text-scale-1200">Explore our other products</h3>
+              <p className="text-base text-scale-1100">
+                Supabase provides all the backend features you need to build a product. You can use
```

Comment:
This sentence feels a little wide — maybe a max-w-2xl on the container or have the second sentence on its own line? 

<img width="755" alt="CleanShot 2022-09-23 at 09 55 35@2x" src="https://user-images.githubusercontent.com/105593/191959695-f85be2af-a678-4144-938a-c17b022435d7.png">